File: C/BEGINNER/1848 Counting Crow.c
#include <stdio.h>
#include <string.h>

void lottery();

int main()
{
    lottery();
    return 0;
}

void lottery()
{
    char crow[10];
    int sum = 0, countShouts = 0;
    while (countShouts != 3)
    {
        fgets(crow, sizeof(crow), stdin);
        if (strcmp(crow, "caw caw\n") == 0)
        {   
            printf("%d\n", sum);
            sum = 0;
            countShouts += 1;
        }
        else if (strcmp(crow, "--*\n") == 0)
        {
            sum += 1;
        }
        else if (strcmp(crow, "-*-\n") == 0)
        {
            sum += 2;
        }
        else if (strcmp(crow, "-**\n") == 0)
        {
            sum += 3;
        }
        else if (strcmp(crow, "*--\n") == 0)
        {
            sum += 4;
        }
        else if (strcmp(crow, "*-*\n") == 0)
        {
            sum += 5;
        }
        else if (strcmp(crow, "**-\n") == 0)
        {
            sum += 6;
        }
        else
        {
            sum += 7;
        }
    }
    return;
}

File: Python/BEGINNER/1848 Counting Crow.py
# SUBPROGRAM
def lottery():
    sum = countCaw = 0
    while countCaw != 3:
        crow = input()
        if crow == "caw caw":
            print("%d" % sum)
            sum = 0
            countCaw += 1
        elif crow == "--*":
            sum += 1
        elif crow == "-*-":
            sum += 2
        elif crow == "-**":
            sum += 3
        elif crow == "*--":
            sum += 4
        elif crow == "*-*":
            sum += 5
        elif crow == "**-":
            sum += 6
        else:
            sum += 7
    return None

# MAIN
lottery()
